Makefile

```diff
@@ -26,6 +26,10 @@ $(DEPS_DIR)/rabbit:
 	git clone https://github.com/rabbitmq/rabbitmq-server.git $@
 	$(MAKE) -C $@ fetch-deps DEPS_DIR="$(abspath $(DEPS_DIR))"
 
+$(DEPS_DIR)/rabbitmq-ct-helpers:
+	rm -rf $(DEPS_DIR)/rabbitmq_ct_helpers
+	git clone https://github.com/rabbitmq/rabbitmq-ct-helpers.git $(DEPS_DIR)/rabbitmq_ct_helpers
```

Comment:
Here you create the directory `$(DEPS_DIR)/rabbitmq_ct_helpers` (with underscores) but the target is `$(DEPS_DIR)/rabbitmq-ct-helpers` (with dashes). Is it expected?

The way it is, the recipe will always be executed because the target is never created.

Reply:
Right! Fixed. Thanks!